tests/load_test.py

```diff
@@ -0,0 +1,117 @@
+import argparse
+import time, asyncio
+from openai import AsyncOpenAI, AsyncAzureOpenAI
+import uuid
+import traceback
+import numpy as np
+
+# base_url - litellm proxy endpoint
+# api_key - litellm proxy api-key, is created proxy with auth
+litellm_client = None
+
+
+async def litellm_completion(args, image_url=None):
+    # Your existing code for litellm_completion goes here
+    try:
+        if image_url:
+            messages = [
+                {
+                    "role": "user",
+                    "content": [
+                        {"type": "image_url", "image_url": {"url": image_url}},
+                        {"type": "text", "text": f"Tell me a story about this image."},
+                    ],
+                },
+            ]
+        else:
+            messages = [
+                {"role": "user", "content": f"Tell me a story about this image."}
+            ]
+
+        start = time.time()
+        response = await litellm_client.chat.completions.create(
+            model=args.model,
+            messages=messages,
+            stream=True,
+        )
+        ttft = time.time() - start
```

Comment:
just to confirm: is it documented anywhere that when this function returns it means first token is ready?